templates/docker-compose.yml.j2

```diff
@@ -96,3 +96,5 @@ services:
 {% for node in go_waku_static_nodes %}
       --staticnode={{ node }}
 {% endfor %}
+      --resource-scaling-memory-percentage={{ go_waku_resource_scaling_memory_percentage | float }}
+      --resource-scaling-fd-percentage={{ go_waku_resource_scaling_fd_percentage | float }}
```

Comment:
Wrong name of flag, fixed in: https://github.com/waku-org/go-waku/commit/0dbe4fd5ff4f647f455b3af08c02910c4e0d8011